### ISSUE TITLE
Runtime Issue with amr-wind/amrex using CUDA

### ISSUE THREAD

**Issue by tchatte3** [AMReX-Codes/amrex#3488]
Hello Amrex dev. team,

We are trying to run simulations using the software amr-wind which is build using ``amrex`` with GPU nodes. The compilation is successful, but at runtime some errors are thrown-up. I believe this is more of an ``amrex`` issue than ``amr-wind`` issue. 

Here are error snapshots
![image](https://github.com/AMReX-Codes/amrex/assets/14302216/1e0dfca4-5f57-4477-a9a9-84952131c347)

I am using CUDA 11.4 and amrex 23.07
![image](https://github.com/AMReX-Codes/amrex/assets/14302216/99dd5df0-cd0e-4dac-826f-794935499752)

Here is the output from Nvidia -smi
![image](https://github.com/AMReX-Codes/amrex/assets/14302216/535db269-c8fe-43a0-8017-3fe51d33e8f3)

Any suggestions to get around this issue?

Best,
Tanmoy

**Comment by WeiqunZhang** [comment]
I don't think this is an issue of amrex. Looks like the cuda installation is broken. You can try to compile and run a simple cuda code to see if it works. If it's available, you can also try to run `/usr/local/cuda/extras/demo_suite/deviceQuery`.

**Comment by asalmgren** [comment]
Closing this issue due to lack of activity.  Please re-open it if you feel there is is still an issue!
